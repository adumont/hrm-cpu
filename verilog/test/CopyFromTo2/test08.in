17 c9
54 74
23 4f
06 99
04 93
52 69
03 5b
24 14
46 28
53 bd
25 89
09 57
29 6b
03 e2
10 9b
09 9b
19 86
16 05
08 5b
56 72
11 75
08 1b
44 e7
56 5f
34 9d
39 a3
51 57
47 1a
21 35
57 fd
30 bb
53 a5
03 cb
16 a7
01 4b
12 f4
12 7a
33 d0
16 e0
40 0c
42 8d
