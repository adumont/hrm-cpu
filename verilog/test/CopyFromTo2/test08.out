c9
74
4f
99
93
69
5b
14
28
bd
89
57
6b
e2
9b
9b
86
05
5b
72
75
1b
e7
5f
9d
a3
57
1a
35
fd
bb
a5
cb
a7
4b
f4
7a
d0
e0
0c
